Kyle

sweetstyle11 opened the pull request.

Commit:
building controllers and views

Commit:
movies added

Commit:
vote links work

Commit:
basic binary battle works

Commit:
Binary Battle works for movies,
I am now trying to make a battle that shows 2 buildings. I can't get local images to display though.

ghost commented:
I'm able to get binary battle to work for text. But I was trying to add another battler that shows two pictures of buildings and asks the user to pick their favorite.  I have a ways to go on this feature but right now I'm having trouble even getting images to show up. Is there more to just using <img> tags? Also, I'm using pictures I've saved locally, should I not do that?

halfghaninne commented:
You can use images that you've saved locally. When you use `<img src ="" ...>` the path that follows `src` is a path to the image file in your project folder, not a path to a controller action.

Please let me know if that clears some things up.

Commit:
images still not showing

ghost commented:
I'm still not able to get the actual image to display in my view. I've tried every variation of the path src="../images/dart.jpg"  or src="images/dart.jpg" etc. And still the only thing that shows up in my view is the alt text that I defined.  I really don't understand why the image itself doesn't show up.

halfghaninne commented:
building_home.erb is located in **web-app__binary-battle/app/views/buildings/** It would take quite a few steps back, then forward again, to get to the correct image file.

What about [starting the path from the project's root folder](http://stackoverflow.com/questions/14489016/how-to-properly-reference-local-resources-in-html)?

ghost commented:
It works if I used use a url for the source, but what if I want to use a picture that I took myself

halfghaninne commented:
Yeah! That's the difference of using locally hosted and externally hosted images! But it's definitely still possible to use locally hosted images. Try the other path variations and let me know how it goes.

Commit:
Gave up on showing local images,
making the building battle with urls from the web.

Commit:
buildings added

Commit:
voting and image display works

Commit:
building battle finished

Commit:
finally got local images to work

Commit:
playing with hidden forms

Commit:
dropping database :(

Commit:
still trying to track matches and match winners

Commit:
beer thirty

Commit:
sorting winning movies

Commit:
A new showdown row is created everytime the homepage is shown,
this causes problems if nobody votes. About to fix that.

Commit:
Still trying to find ties, ok

Commit:
i still cant figure out ties ok

Commit:
This isn't working, switching my ranking system

Commit:
I can count and sort by wins,
but I can't associate a movie with both its wins and loses at the same time.

Commit:
Changed Building counter to track wins and losses
also added local images to the database.

Commit:
new day, going to implemnt RPI ranking system

Commit:
can find winning percentage

Commit:
Calculate a movie's winning percentage and store in database,
now working on writing method to calculate oponent's wining percentage. This is all part of a plan to calculate the Ratings Percent Index of each movie. This theoretically would sort movies even if there are ties.

Commit:
owp almost there

Commit:
oponents win average works i thing

Commit:
oowp now works i thing

Commit:
i think this all works, something has to be wrong

Commit:
rpi ranking system works, sort of)

ghost commented:
This isn't so much a question as more just checking in. I got a little consumed by trying to find a way to rank my movies without ties.  This proved to be really challenging and in the spirit of the upcoming March Madness I decided to try and implement the same ranking system that the NCAA uses to rank basketball teams. This works, but I realized after making it that there is huge flaw because the NCAA ranks teams that have all played the same number of games, and binary battles are completely random. Anyways, like I said, just checking in.
